docs/changelog.md

```diff
@@ -28,6 +28,11 @@ permalink: /changelog/
   * See component's [README](https://github.com/mozilla-mobile/android-components/blob/master/components/service/firefox-accounts/README.md) for detailed description of the new API.
   * As part of these changes, token caching issue has been fixed. See [#3579](https://github.com/mozilla-mobile/android-components/pull/3579) for details.
 
+* **concept-engine**, **browser-engine-gecko(-beta/nightly)**.
+  * Added `TrackingProtectionPolicy.CookiePolicy()` to indicate how cookies should behave for a given `TrackingProtectionPolicy`.
+  * Now `TrackingProtectionPolicy.select` allows you to indicate a `TrackingProtectionPolicy.CookiePolicy()`, if you don't specified any the `TrackingProtectionPolicy.CookiePolicy.ACCEPT_NON_TRACKERS` will be assigned.
```

Comment:
allows you to specify ... ? If not specified, `CookiePolicy.ACCEPT_NON_TRACKERS` will be used.

Reply:
Fixed!